rest-service/manager_rest/rest/resources_v2/events.py

```diff
@@ -50,7 +50,7 @@ class Events(resources_v1.Events):
         notes='Returns a list of events for optionally provided filters'
     )
     @authorize('event_list')
-    @rest_decorators.marshal_events
+    @rest_decorators.marshel_list_response
```

Comment:
```suggestion
    @rest_decorators.marshal_list_response
```
ditto other occurrences